param_sweeps/driver.py

```diff
@@ -129,7 +129,7 @@ def get_lookup(self):
         iterations = list(itertools.product(*sets.values()))
         for iteration in iterations:
             param_uuid = SweepDriver.uuid_from_params(iteration)
-            lookup[param_uuid] = dict(zip(sets.keys(), iteration))
+            lookup[param_uuid] = dict(zip(sets.keys(), iteration, strict=False))
```

Comment:
**question:** should use `strict = True` ?

@benk-mira, is the size of `sets.keys()` expected to be equal to the size of iteration?